docs/UserGuide.md

```diff
@@ -41,146 +38,415 @@ By : `Team SE-EDU`  &nbsp;&nbsp;&nbsp;&nbsp; Since: `Jun 2016`  &nbsp;&nbsp;&nbs
 > * Items with `...` after them can have multiple instances.
 > * Parameters can be in any order.
 
-### 2.1. Viewing help : `help`
+### 2.1. Saving the data
+
+By default, task manager data is saved in the hard disk automatically after any command that changes the data.
+There is no need to save manually.
+
+### 2.2. Viewing usage instructions : `help`
 
 Format: `help`
 
 > Help is also shown if you enter an incorrect command e.g. `abcd`
 
-### 2.2. Adding a person: `add`
+### 2.3. Adding a task with no specified completion date: `add`
+
+Adds a task to the task manager
+
+Format: `add DESCRIPTION`
+
+> Tasks will be added to the task manager with no due date or duration.
+
+Examples:
+
+* `add write essay`
+
+### 2.4. Adding a task with a due date: `add ... by ...`
+
+Adds a task with a due date to the task manager
+
+Format: `add DESCRIPTION by DUE_DATE`
+
+> Tasks with a due date will be added to the task manager.
+
+Examples:
+
+* `add write essay by 13/7/17`
+
+### 2.5. Adding a task with a start date: `add ... from ...`
 
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
+Adds a task with a start date to the task manager
 
-> Persons can have any number of tags (including 0)
+Format: `add DESCRIPTION from START_DATE`
+
+> Tasks with a start date will be added to the task manager.
+
+Examples:
+
+* `add write essay from 13/7/17`
+
+### 2.6. Adding a task with a duration: `add ... from ... to ...`
+
+Adds a task to the task manager
+
+Format: `add DESCRIPTION from START_DATE to END_DATE`
+
+> Tasks will be added to the task manager with a duration.
+
+Examples:
+
+* `add write essay from 13/7/17 to 15/7/17`
+
+### 2.7. Adding a task with tags: `add ... #TAGNAME ...`
+
+Adds tags to the task
+
+Format: `add DESCRIPTION #TAGNAME`
+
+> Tasks will be added to the task manager with a tags.
 
 Examples:
 
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe t/friend e/betsycrowe@gmail.com a/Newgate Prison p/1234567 t/criminal`
+* `add write essay from 13/7/17 to 15/7/17 #school #homework`
+* `add write essay from 13/7/17 #school #homework`
+* `add write essay by 15/7/17 #blogging`
+* `add write essay #blogging`
+
 
-### 2.3. Listing all persons : `list`
+### 2.8. Listing all tasks : `list`
+
+Shows a list of all tasks in the task manager.
 
-Shows a list of all persons in the address book.<br>
 Format: `list`
```

Comment:
I guess you can skip this since there is no another format.  Likewise for `list done` and `list undone`.  Or maybe combine all three under `list` so that now you can show the different formats?